### DISCUSSION TITLE
How to pass something to a different Window using CommunityToolkit?

### DISCUSSION THREAD

**Discussion by vo1s** [AvaloniaUI/Avalonia#14605] [Q&A] [answered]
For example I have a Bitmap I want to pass into a different Avalonia Window **(from one ViewModel to another)** using CommunityToolkit to display it in it. What pattern should use?

**Comment by thevortexcloud** [comment]
Just pass the bitmap to your other VM. You can do that either by setting a property, using the constructor, of using events. I don't think CommunintyToolkit will make much of a difference here. 

You can technically do it on your view side too, it's just not a good idea since the view is Avalonia specific and should not be where your main application logic is.

**Comment by workgroupengineering** [comment, marked as answer]
Use [messenger](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/messenger)
